save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/TestExecutionDetailsView.kt

```diff
@@ -112,7 +112,7 @@ fun testExecutionDetailsView() = fc<Props> {
     val (testResultDebugInfo, setTestResultDebugInfo) = useState<TestResultDebugInfo?>(null)
 
     // fixme: after https://github.com/saveourtool/save-cloud/issues/364 can be passed via history state to avoid requests
-    useRequest(arrayOf(status, testResultDebugInfo, executionId, testResultLocation), isDeferred = false) {
+    useRequest(arrayOf(params), isDeferred = false) {
```

Comment:
Why do we use `arrayOf` here? Far as I remember `dependencies` is `vararg` of `dynamic`

Reply:
Oh ok